### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by vicoliveir** [community/community#43128] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Perfil Público

### Body

Como eu crio um perfil público na minha página inicial do GitHub?

**Comment by artie93** [comment, marked as answer]
Oi @vicoliveir ! 
É simples: cria um repositório novo com um arquivo README.md que tenha exatamente o nome do seu usuário (vicoliveir). Esse vai ser o arquivo que vai aparecer como seu perfil público aqui na comunidade. Existem algumas coisas que vc pode adicionar pra deixar o perfil bacana. Dá uma olhada no meu perfil depois e fica à vontade se quiser copiar algo de lá pro seu perfil.

Espero ter ajudado :)

**Reply by vicoliveir** [reply]
Oi @artie93 !
Adorei seu perfil. Obrigado pelo feedback, vou fazer algumas mudanças em breve.
